test/int/core-v2/typeUtils.ts

```diff
@@ -5,6 +5,8 @@
 /**
  * Type utilities to construct derived types from the original types, rather than have to manually write them
  */
+
+export type MakeAllRequired<T> = { [P in keyof T]-?: T[P]; };
```

Comment:
TS has a builtin type helper for this, `Required` I think it's the same

Reply:
Good catch. I thought I remembered there being a built-in one that did the same thing but couldn't find it for whatever reason.